### ISSUE TITLE
Metasploit-framework crashed in latest upgrade for Kali Rolling-2016.1

### ISSUE THREAD

**Issue by hshivam00** [bundler/bundler#5018]
Hi everyone,
Recently I upgraded my Kali Linux which was previously a version Kali-Rolling and after an upgrade I got Kali-2016.1. However, after a successful upgrade, I tried to run "msfconsole" to launch a Metasploit-Framework. However, it wasn't a successful startup and it crashed. I then removed it completely and then downloaded and set up a whole new master version of metasploit framework from GitHub. After performing all necessary set ups, I again tried to run "msfconsole" and had some new issues. I have given a snapshot of what I got in terminal. If anyone have got any solution or anything, please revert back with a comment. Thanks!

```
root@kali:~# msfconsole
Your bundle is locked to rake (11.2.2), but that version could not be found in any of the sources listed in your Gemfile. If you haven't changed sources, that means the author of rake (11.2.2) has removed it. You'll need to update your bundle to a different version of rake (11.2.2) that hasn't been removed in order to install.
Run `bundle install` to install missing gems.
root@kali:~# 

```

**Comment by chrismo** [comment]
If it's possible for you to reduce to a simpler reproduction case, that would be helpful to us - otherwise it'll probably take us longer to get around to trying to recreate from within the metasploit-framework.

**Comment by BytesDirect** [comment]
Hi.

I was thinking, could the problem be with Rake (11.2.2)?

**Comment by BytesDirect** [comment]
Or RVM? as first I used rbenv to install metasploit, however I was getting issues with it. And when I run this command: `user@user-pc-SATELLITE-C55-A:/opt/metasploit-framework$ bundle exec gem list` I get this:
`rbenv: version ``2.3.1'` `is not installed ``(set by /opt/metasploit-framework/.ruby-version)`
`user@user-pc-SATELLITE-C55-A:/opt/metasploit-framework$ `

**Comment by BytesDirect** [comment]
However, I did uninstall rbenv with `sudo apt-get remove rbenv`, so I'm just completely lost now.

Ty

**Comment by hshivam00** [comment]
Hi everyone,
At last, I did configured metasploit-framework with successfully working environment. Actually the problems were with Rake-11.2.2 and much more. However, I did uninstall it 7-8th time, and then, instead of installing it manually through termial, I preferred to do so using other install methods. As part of an installation process, I first downloaded Metasploit-Framework installer which comes with an extension as .run file type. Hence, installing this file wasn't much hectic for me and that gave me an installation interface. It is working now with regular updates.
Thanks all for your responses!
